pkg/kubectl/stop.go

```diff
@@ -78,19 +78,26 @@ func (reaper *ReplicationControllerReaper) Stop(namespace, name string) (string,
 	return fmt.Sprintf("%s stopped", name), nil
 }
 
-func (reaper *PodReaper) Stop(namespace, name string) (string, error) {
+func (reaper *PodReaper) Stop(namespace, name string, gracePeriodSec int) (string, error) {
 	pods := reaper.Pods(namespace)
 	_, err := pods.Get(name)
 	if err != nil {
 		return "", err
 	}
-	if err := pods.Delete(name); err != nil {
-		return "", err
+	if gracePeriodSec >= 0 {
+		if err := pods.DeleteGracefully(name, uint64(gracePeriodSec)); err != nil {
```

Comment:
The code uses int, int64, and uint64 in different places. Please be consistent. And unsigned is dubious if you're going to use -1 for anything.